### ISSUE TITLE
scrolling causes issues (code renderer CPU spike) when using a screenreader (like vimac)

### ISSUE THREAD

**Issue by dennis-damico** [microsoft/vscode#110764]
- VSCode Version: 1.51.1
- OS Version: macOS Catalina 10.15.7

Steps to Reproduce:

1. Open VSCode
2. Enable vimac
3. Trigger vimac in vscode (not necessary but will guarantee the issue be reproduced)
4. Open a file in vscode an scroll / move cursor around. You will see the window lag and CPU spike

Does this issue occur when all extensions are disabled?: Yes

Note: I'm not entirely sure that vimac is a screenreader though i believe it is because when i first enabled it vscode displayed a message (only time ive ever seen this) mentioning `Controls whether the editor should run in a mode where it is optimized for screen readers` (setting this to on does not help btw)

**Comment by dennis-damico** [comment]
If anyone looks further into this may be worth looking [here](https://github.com/dexterleng/vimac/issues/264) as well in case it's somehow related to vimac implementation (though im not sure how vimac would cause high cpu on the code renderer process)

**Comment by isidorn** [comment]
Thanks for the feedback. Do you see the same when you use vimac and chrome, for example this page https://bocupp-microsoft.github.io/sample-code/textarea/setselectionrange-performance.html

If you see slowness then could you please file an issue against Chrome?
If not, do you see slowness with other Electron apps and vimac?

**Comment by dennis-damico** [comment]
I went to the link you sent and did not experience any issues there. Slack and Bitwarden (though bitwarden is pretty minimal so maybe a bad comparison) both run fine as well (cannot tell a difference when extension is enabled vs disabled)

**Comment by dexterleng** [comment]
> Note: I'm not entirely sure that vimac is a screenreader

Yes, It uses AXManualAccessibility to be able to read the element tree.

**Comment by dexterleng** [comment]
This is likely a chromium accessibility performance issue, not a VSCode one.

See the original discussion on adding AXManualAccessibility:
https://github.com/electron/electron/issues/7206

**Comment by isidorn** [comment]
@dexterleng yes sounds like Chromium. Best would be if we could get reproducable steps for Chrome, so we can file an issue in the Chromium repository.
